### PR TITLE
Fix CLA rendering and improve aperture-operator handling

### PR DESCRIPTION
**Pull request by sbienkow-ninja**
This change is [Reviewable](https://reviewable.io/reviews/fluxninja/aperture/251)

### CODE REVIEW

File: manifests/charts/aperture-operator/values.yaml [hunk 1]
```diff
@@ -355,9 +355,9 @@ agent:
   ## @param agent.image.pullSecrets Agent image pull secrets
   ##
   image:
-    registry: ~
-    repository: ~
-    tag: ~
+    registry: docker.io/fluxninja
```
Comment (hdkshingala) [inline manifests/charts/aperture-operator/values.yaml:358]: This values are configured as default in the operator so we can skip them for Agent and Controller section.

Comment (sbienkow-ninja) [reply manifests/charts/aperture-operator/values.yaml:358]: Oh, ok. When I was testing it tomorrow, it tried to download image from the wrong registry, but it's possible that it was just the operator. I wonder if it'd be worth generating the `values.yaml` here based on the operator defaults - so they are easier to view by users?

Comment (hdkshingala) [reply manifests/charts/aperture-operator/values.yaml]: If we add all the defaults here, the generated YAML for CR will be long and user may feel that the CR needs all the values always. Thinking this, I kept only the required ones. What's your thoughts on that?

Comment (sbienkow-ninja) [reply manifests/charts/aperture-operator/values.yaml]: That's also a good point. I'm fine either way.

Comment (hdkshingala) [reply manifests/charts/aperture-operator/values.yaml]: Okay.. let's continue with the minimum required ones for now and will add all if we see confusion around that later.

**Pull request merged**